src/Jackalope/Node.php

```diff
@@ -1462,17 +1478,6 @@ public function confirmSaved()
     }
 
     /**
-     * In addition to calling parent method, tell all properties
-     */
-    public function setPath($path, $move = false)
```

Comment:
I've understood my implementation in a way that `loadProperty` is called for every property (since all the `_setProperty` calls do not instantiate properties anymore). And I've moved the `setPath` call there, is there a scenario where this would not work?